agent/http.go

```diff
@@ -29,6 +29,7 @@ import (
 	"github.com/hashicorp/go-cleanhttp"
 	"github.com/mitchellh/mapstructure"
 	"github.com/pkg/errors"
+	"github.com/ryboe/q"
```

Comment:
You probably didn't mean to leave this in.

Reply:
Indeed, I fixed that - the lib changed owner recently so my usual precommit hook didn't catch it. Apparently didn't push the fix yet although I thought I had.